app/models/report_composer.rb

```diff
@@ -99,19 +141,9 @@ def to_params
   end
 
   def valid?
-    super & @input_values.map { |_, input_value| input_value.valid? }.all?
-  end
-
-  def errors
```

Comment:
what is the reasoning for removal? note that it `dup`ped the errors object before, I suppose there was a reason for that

Reply:
It was exactly the issue, because validators were adding errors to the dupped object and throwing them on next call to `errors`, I believe that `dup` were for not duplicating the input_values error on every call, wich is moved to the `valid?` call, where the current errors are cleared beforehand, thus no duplication will take place and even seems bit cleaner to me.

Reply:
Forgot to mention, that the `dup` made adding errors on main model impossible.